honggfuzz/tag

```diff
@@ -0,0 +1 @@
+1.0.0-honggfuzz1.9
```

Comment:
shouldn't this be `1.9`?

Reply:
nvm, I interpreted the pattern wrong lol, will make the change

Reply:
Fixed!